docs/config.md

```diff
@@ -4,11 +4,10 @@
 
 | Environment variable | Description | Default |
 |-|-|-|
-| `OTEL_DOTNET_AUTO_HOME` | Installation location. | `true` |
+| `OTEL_DOTNET_AUTO_HOME` | Installation location. |  |
 | `OTEL_DOTNET_AUTO_ENABLED` | Enables the tracer. | `true` |
 | `OTEL_DOTNET_AUTO_INCLUDE_PROCESSES` | Names of the executable files that the profiler can instrument. Supports multiple comma-separated values, for example: `MyApp.exe,dotnet.exe`. If unset, the profiler attaches to all processes by default. |  |
 | `OTEL_DOTNET_AUTO_EXCLUDE_PROCESSES` | Names of the executable files that the profiler cannot instrument. Supports multiple comma-separated values, for example: `ReservedProcess.exe,powershell.exe`. The list is processed after `OTEL_DOTNET_AUTO_INCLUDE_PROCESSES`. If unset, the profiler attaches to all processes by default. |  |
-| `OTEL_DOTNET_AUTO_AZURE_APP_SERVICES` | Set to indicate that the profiler is running in the context of Azure App Services. | `false` |
```

Comment:
This is an interesting one, because it is used to toggle on some process exclusion logic that applies to azure environments. Maybe some of these things should not be called out here, but belong in some other document? Maybe it makes sense to have a document related to azure at some point?

Reply:
Perhaps we should track the "non-documented" or "internal" env vars in different document so we can keep the record of the intended usage and later we move them to the proper markdown giving it the proper visibility.

Reply:
I suggest moving those to a `dev` or `internal` folder within `docs`.

Reply:
Good idea, added to `/docs/internal/config.md`